ui/v2.5/src/locales/pt-BR.json

```diff
@@ -0,0 +1,601 @@
+{
+  "actions": {
+    "add": "Adicionar",
+    "add_directory": "Adicionar diretório",
+    "add_entity": "Adicionar {entityType}",
+    "add_to_entity": "Adicionar em {entityType}",
+    "allow": "Permitir",
+    "allow_temporarily": "Permitir temporariamente",
+    "apply": "Aplicar",
+    "auto_tag": "Auto tag",
+    "backup": "Backup",
+    "cancel": "Cancelar",
+    "clean": "Limpar",
+    "clear_back_image": "Limpar imagem de fundo",
+    "clear_front_image": "Limpar imagem frontal",
+    "clear_image": "Limpar imagem",
+    "close": "Fechar",
+    "create": "Criar",
+    "create_entity": "Criar {entityType}",
+    "create_marker": "Criar marcador",
+    "created_entity": "Criar {entity_type}: {entity_name}",
+    "delete": "Apagar",
+    "delete_entity": "Apagar {entityType}",
+    "delete_file": "Apagar arquivo",
+    "delete_generated_supporting_files": "Apagar arquivos gerados de suporte",
+    "disallow": "Não permitir",
+    "download": "Download",
+    "download_backup": "Download backup",
+    "edit": "Editar",
+    "export": "Exportar…",
+    "export_all": "Exportar tudo…",
+    "find": "Encontrar",
+    "from_file": "Do arquivo…",
+    "from_url": "Da URL…",
+    "full_export": "Exportação completa",
+    "full_import": "Importação completa",
+    "generate": "Gerar",
+    "generate_thumb_default": "Gerar thumbnail padrão",
+    "generate_thumb_from_current": "Gerar thumbnail do atual",
+    "hash_migration": "migrar hash",
+    "hide": "Esconder",
+    "import": "Importar…",
+    "import_from_file": "Importar do arquivo",
+    "merge": "Unir",
+    "merge_from": "Unir do",
+    "merge_into": "Unir em",
+    "not_running": "não realizado",
+    "overwrite": "Sobrescrever",
+    "play_random": "Tocar aleatório",
+    "play_selected": "Tocar selecionado",
+    "preview": "Previsualizar",
+    "refresh": "Atualizar",
+    "reload_plugins": "Recarregar plugins",
+    "reload_scrapers": "Recarregar scrapers",
+    "remove": "Remover",
+    "rename_gen_files": "Renomear arquivos gerados",
+    "rescan": "Reescanear",
+    "reshuffle": "Reembaralhar",
+    "running": "rodando",
+    "save": "Salvar",
+    "save_filter": "Salvar filtro",
+    "scan": "Escanear",
+    "scrape_with": "Scrape com…",
+    "search": "Buscar",
+    "select_all": "Selecionar todos",
+    "select_none": "Selecionar nenhum",
+    "selective_auto_tag": "Auto Tag seletivo",
+    "selective_scan": "Scaneamento seletivo",
+    "set_as_default": "Aplicar como padrão",
+    "set_back_image": "Imagem de fundo…",
+    "set_front_image": "Imagem frontal…",
+    "set_image": "Aplicar imagem…",
+    "show": "Mostrar",
+    "skip": "Pular",
+    "tasks": {
+      "clean_confirm_message": "Tem certeza de que quer limpar? Isto deletará as informações do banco de dados e conteúdos gerados de todas as cenas e galerias que não são mais encontradas no sistema.",
```

Comment:
Could change "deletará" here to "irá apagar"

Reply:
Similar thing to lines below

Reply:
What I said is "Irá apagar" changing deletar to apagar, and the lines below where "deletado" could be "apagado"